### ISSUE TITLE
Maybe we should create a collective blog over this domain?

### ISSUE THREAD

**Issue by Jabher** [brainfucker/nodejs.ru#3]
I do not know what will happen with that domain and what others expect but what I see (in general):
- Habr is slowly going dead
- We have no big node.js resources in Russian
- local projects alike frontender.info are becoming rapidly popular
- some kind of feed (at least with news) will be here, probably, anyway.

Let's talk about what platform to use/create and what we want to make here.

**Comment by brainfucker** [comment]
Согласен, блог необходим, при этом мне очень симпатизирует идея добавления статей через github в markdown, тк благодаря высокому порогу входа этот подход, позволит полностью избежать модераторов итд + в случае если с сайтом что-то случится  любой сможет поднять зеркало.

**Comment by Jabher** [comment]
не уверен, что отказ от модерации это хорошая идея, лучше уж самоорганизующееся сообщество, более или менее. Механизм pull requests на первое время может помочь, а дальше нужно думать.

**Comment by brainfucker** [comment]
Сейчас пожалуй не очень ясно, зависит от сообщества которое сформируется, можно хоть хабр в плане функционала сделать, но сначала нужно собрать интересное сообщество.. Пока я бы просто собирал статьи на markdown, посмотрим, буду принимать любые улучшения контента или движка

**Comment by Gormartsen** [comment]
Привет всем!
Я заинтересован в создании сайта сообщества.

На текущий момент есть нароботки в этом плане по Drupal.ru

На сейчас могу банально предложить использовать Drupal, для создания сайта сообщества с уже наработаными механизмами.

Я сейчас паралельно занимаюсь изменением самого Drupal.ru и эти работы можно совместить.

Кто что думает?

**Comment by brainfucker** [comment]
Идеологически очень хочется чтобы сайт о node.js был написан на node.js. В данном случае не думаю что проблема в движке, скорее нужен материал, статьи

**Comment by Gormartsen** [comment]
С идеологической стороны - понимаю.
Но это займет ооооочень много человеко-часов - сделать комунити платформу.

Можно сейчас начать с Drupal, мне это займет поставить и настроить пару дней.
Цветовую гамму только поменять на  зелено черно белую.

А когда подтянется народ, уже можно будет быстрее написать свой вариант комунити на nodejs

**Comment by Jabher** [comment]
Так Ghost же есть на ноде. 

Вопрос скорее в том, что писать в блог.

**Comment by Gormartsen** [comment]
Надо не блок делать, а больше аля форум. Что б народ мог задавать вопросы, отвечать.
Подсветка кода что б была.
Цитирования, аватарки, подписи, контакты.
Профайл страница.

Авторизация.
Таксономия (теги)

Группировка постов по тегам.

Разделы промо.

Это так. сумбурно сходу написал.

**Comment by Jabher** [comment]
что-то а-ля хабр?

**Comment by Gormartsen** [comment]
хабр имеет свою специфику. 
Я больше вот о таких вещах:
http://www.drupal.ru/username/gor - профайл, где видно контакты. можно послать приватное сообщение юзеру (внутренний месенджер)
http://www.drupal.ru/forum/site (раздел форума)
http://www.drupal.ru/taxonomy/term/2708 (материалы по тегу)
http://www.drupal.ru/alttracker (последние материалы на сайте)
http://www.drupal.ru/node/126483 (пример подсветки)

ну и само собой комментарии, с bbcode, вставкой и аплоадом скриншотов, цитирование, оповещение о новых комментариях.

Это на вскидку. 

Я сейчас на drupal.ru планирую много чего нового внедрить заточенного на развитие сообщества:
- галерея работ (сайтов) в ru зоне
- профайлы компаний что сервисы предлагают
- более продвинутые профили юзеров (с работами, статистикой активности, гитхаб активностью)
- централизованный календарь drupalcafe, кемпов, дринкапов (когда народ в баре собирается и за пивом общается на около друпал темы).

и многое другое. 

Вообщем если в эту сторону развивать сходу сайт, можно получить серьезный буст комунити.

**Comment by Jabher** [comment]
боже, только не бибикод.

**Comment by Gormartsen** [comment]
bbcode имеет свои плюсы. Например легко вставлять ссылки, картинки, стили кода, цитирование, отступы.
Его достаточно много пользователей знает.
Но если заморачиваться, то можно и mardown: https://www.drupal.org/project/markdown

**Comment by cassln** [comment]
@Gormartsen прочитайте второе сообщение в этом issue

**Comment by Jabher** [comment]
давайте тогда на [prose.io](http://prose.io) что-нибудь поднимем. Например blog.nodejs.ru хотя бы там развернем. Это честный статический генератор онлайн все-таки.

**Comment by Gormartsen** [comment]
@cassln Читал. Это хорошо для просто статей, но не поможет создать комунити.
Если же говорить про комунити, то общение в github issue - это не способ его создания, по моему мнению.

**Comment by ewnd9** [comment]
Можно лучше начать с создания gitter или slack чата?

**Comment by Gormartsen** [comment]
++ gitter

**Comment by Jabher** [comment]
ну, я поднял nodejs-ru.slack.com
@brainfucker , мне бы домен для открытых инвайтов, насильно приглашать всех подряд не хочется.

**Comment by ewnd9** [comment]
@Jabher можно же на heroku поднять приглашалку, а домен потом прикрутить

**Comment by Jabher** [comment]
@ewnd9 тут дел на день-два. а будет удобно и полноценно. я один черт на хероку собирался это поднять, если честно, но домен привязать - приятно.

**Comment by ewnd9** [comment]
@Jabher "тут дел на день-два" есть готовые решения, например https://github.com/rauchg/slackin

**Comment by Jabher** [comment]
@ewnd9 да я знаю. просто реально это съест минут 20, но задача закроется дня через два из-за времени реакции всех подряд.

**Comment by Gormartsen** [comment]
Ниразу не пользовался slack. Зайти не могу, говорит инвайт должен кто то дать. Кто нибудь даст инвайт?)
Или подождать пока сделаете домен приглашалаку?

**Comment by brainfucker** [comment]
Расскажите пожалуйста подробнее, что а страничка для слека нужна

**Comment by vlaim** [comment]
+1 за slack  
@brainfucker, глянь ссылку выше, что @ewnd9 кинул.  там будет страничка, которая будет генерить инвайты в канал :)

**Comment by Jabher** [comment]
@ewnd9 да ты прав оказался. я и не знал, что теперь на хероку одной кнопкой можно все разворачивать.
В общем, сделал я ссылку для регистрации - https://rocky-badlands-5276.herokuapp.com/

@brainfucker slack априори спроектирован как закрытая система с инвайтами, публичная страничка нужна для автоинвайтов.
